### DISCUSSION TITLE
Flutter stripe error

### DISCUSSION THREAD

**Discussion by naseerx** [flutter-stripe/flutter_stripe#804] [Q&A]
E/flutter (20651): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: PlatformException(flutter_stripe initialization failed, The plugin failed to initialize:
E/flutter (20651): Your theme isn't set to use Theme.AppCompat or Theme.MaterialComponents.

facing this issue from morning but didn't get any solution trying all of the above steps but still same 

**### Styles.xml**
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <!-- Theme applied to the Android Window while the process is starting when the OS's Dark Mode setting is on -->
    <!-- TODO document the necessary change -->
    <style name="LaunchTheme" parent="Theme.AppCompat.Light.NoActionBar">
        <!-- Show a splash screen on the activity. Automatically removed when
             Flutter draws its first frame -->
        <item name="android:windowBackground">@drawable/launch_background</item>
    </style>
    <!-- Theme applied to the Android Window as soon as the process has started.
         This theme determines the color of the Android Window while your
         Flutter UI initializes, as well as behind your Flutter UI while its
         running.

         This Theme is only used starting with V2 of Flutter's Android embedding. -->
    <style name="NormalTheme" parent="Theme.MaterialComponents">
        <item name="android:windowBackground">?android:colorBackground</item>
    </style>
</resources>
```
_Flutter (Channel stable, 2.10.3, on Microsoft Windows [Version 10.0.19044.1766], locale en-US)_

**flutter_stripe: ^3.1.0**

environment:
  sdk: ">=2.16.1 <3.0.0"

        compileSdkVersion flutter.compileSdkVersion
        minSdkVersion 21
        targetSdkVersion 28

**Comment by remonh87** [comment]
I would check https://github.com/flutter-stripe/flutter_stripe/discussions/538 . Make sure you applied the Android instructions in our readme and did that for all themes (both nice and day) and flavors if you have any.

**Reply by naseerx** [reply]
i have applied all the instruction in readmi buy facing the issue

**Comment by remonh87** [comment]
@oliveyew  this is a forum for helping out people not selling stuff
